src/event-manager/event-channel-client.ts

```diff
@@ -72,16 +72,14 @@ export default class EventChannelClient {
       });
       this._wsClient.on('open', () => {
         this._isConnected = true;
+        this.startHeartbeatTimer(connectionOption.heartbeatIntervalMs || (15000 + 1000)); // NOTE: This time must be longer than blockchain event handler heartbeat interval.
```

Comment:
Could you define a constant for the default value? e.g. DEFAULT_HEARTBEAT_INTERVAL_MS?

Reply:
Looks good! Done. Additionally, `DEFAULT_HANDSHAKE_TIMEOUT_MS` was defined.